src/index.html

```diff
@@ -7,6 +7,70 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="content">
+      <article class="content__calendar calendar calendar--month-length--29 calendar--start-day--sun">
+        <div class="calendar__day"></div>
+
+        <div class="calendar__day"></div>
+
+        <div class="calendar__day"></div>
```

Comment:
No need of empty lines between single-line siblings.
```suggestion
        <div class="calendar__day"></div>
        <div class="calendar__day"></div>
        <div class="calendar__day"></div>
```